### DISCUSSION TITLE
Is it faster to use the fetch API or supabase API to retrieve data from a supabase table?

### DISCUSSION THREAD

**Discussion by tryggs** [supabase/supabase#7739] [Questions] [answered]
the title sums up my question pretty good :)

**Comment by GaryAustin1** [comment, marked as answer]
If you are talking the client code like supabase-js as the supabase API versus just making direct calls with http requests with fetch...

The only difference should be a bit of javascript overhead to reformat the "easier to read" js calls into a fetch operation.  The client code will also format your headers.  

There should be no difference in the performance once the http request is issued from either method.
